### ISSUE TITLE
[NuGet.org Bug]: A11y_NuGet Service_Upload Package_Custom_Screenreader : Narrator is not announcing error message.

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/NuGetGallery#8923]
### Impact

It's more difficult to complete my work

### Describe the bug

Source: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1438007

### Repro Steps

1. Hit the URL: https://www.nuget.org/ 
1. Tab navigate to "Sign in" control and hit enter.
1. Tab navigate to "Sign in with Microsoft" control and hit enter.
1. Select “Upload Package” from the Profile Dropdown and hit enter.
1. Tab navigate to "Custom" control and hit enter.
1. Check After press enter on preview button then getting error message and that error message narrator is announcing or not. 

After press enter on "Preview" button then getting error message and that error message narrator is not announcing. 

Only screenreader user will difficult to know the error message.

### Expected Behavior

Narrator should be announce error message.
